README.md

```
@@ -1,86 +1,62 @@
-*Add caption explaining what this shows*
-
-![Screenshot3](Add screenshot 3 here with proper name)
-*Add caption explaining what this shows*
+![image](https://github.com/user-attachments/assets/d36d57f4-0d93-4147-8f11-f3ff77d57366)
 
 # Diagrams
-![Workflow](Add your workflow/architecture diagram here)
-*Add caption explaining your workflow*
-
-For Hardware:
+https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.researchgate.net%2Ffigure%2FText-To-Speech-system-architecture_fig1_241025907&psig=AOvVaw3PsFTcCHsudeb9lJAca1yb&ust=1737957205954000&source=images&cd=vfe&opi=89978449&ved=0CBQQjRxqFwoTCNDL16XZkosDFQAAAAAdAAAAABAE
```

Markdownlint (0.37.0)

Bare URL used
null

(MD034, no-bare-urls)